recipes/genome-properties/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "2.0.1" %}
+
+package:
+  name: genome-properties
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('genome-properties', max_pin="x.x") }}
+
+source:
+  url: https://github.com/EBI-Metagenomics/genome-properties/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 628428e7ed2b74c16703fcf8e825e84d4cbcbadf4dc61dac6c64e89f35694ecc
+
+requirements:
+  build:
+    - make
+    - bioconda::perl-bioperl
+    - perl-app-cpanminus
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Why do you need a compiler? Where do you use them?

Reply:
These were necessary to build the two perl packages the `build.sh` script installs with `cpanm`, so if they were on bioconda they wouldn't be necessary